articles/azure-functions/functions-bindings-timer.md

```diff
@@ -287,7 +287,9 @@ Expressed as a string, the `TimeSpan` format is `hh:mm:ss` when `hh` is less tha
 |---------|---------|
 |"01:00:00" | every hour        |
 |"00:01:00"|every minute         |
-|"24:00:00" | everyday        |
+|"24:00:00" | every 24 days        |
+|"1:00:00:00:00" | everyday |
+|"1" | everyday |
```

Comment:
I'd prefer to not include this shorthand in the docs. Being explicit in for values like these makes code more maintainable. Please remove this line.

Reply:
```suggestion
```

@craigshoemaker - Agreed. Conventional notation should be followed. Suggested removal in this comment to try to help the author.